includes/pattern.inc

```diff
@@ -181,11 +181,18 @@ function oe_bootstrap_theme_preprocess_pattern_modal(array &$variables): void {
  * Implements hook_preprocess_HOOK().
  */
 function oe_bootstrap_theme_preprocess_pattern_link(array &$variables): void {
-  if (isset($variables['icon']) && !is_array($variables['icon'])) {
+  if (!isset($variables['icon'])) {
+    return;
+  }
+
+  // Convert to array when only the icon name has been passed.
+  if (!is_array($variables['icon'])) {
     $variables['icon'] = [
       'name' => $variables['icon'],
-      'path' => $variables['bcl_icon_path'],
     ];
   }
 
+  $variables['icon'] += [
+    'path' => $variables['bcl_icon_path'],
```

Comment:
Question: If icon pattern has already hardcoded the path with the bcl_icon_path, do we really need this?